examples/GlobalInputHandler/GlobalInputHandler.ino

```diff
@@ -1,8 +1,8 @@
 #include <Homie.h>
 
-HomieNode lightNode("light", "switch");
+HomieNode lightNode("light", "Light", "switch");
 
-bool globalInputHandler(const HomieNode& node, const String& property, const HomieRange& range, const String& value) {
+bool globalInputHandler(const HomieNode& node, const HomieRange& range, const String& property, const String& value) {
```

Comment:
Ranges are not part of Homie 3?

Reply:
Ranges are how Node arrays are called in homie-esp8266, it's just an interval attribute name.